src/Akka.Persistence.PostgreSql/Journal/PostgreSqlDbBatchQueryExecutor.cs

```diff
@@ -0,0 +1,124 @@
+﻿//-----------------------------------------------------------------------
+// <copyright file="PostgreSqlQueryExecutor.cs" company="Akka.NET Project">
+//     Copyright (C) 2009-2016 Lightbend Inc. <http://www.lightbend.com>
+//     Copyright (C) 2013-2016 Akka.NET project <https://github.com/akkadotnet/akka.net>
+// </copyright>
+//-----------------------------------------------------------------------
+
+using Akka.Actor;
+using Akka.Persistence.Sql.Common.Journal;
+using Akka.Serialization;
+using Akka.Util;
+using Newtonsoft.Json;
+using Npgsql;
+using NpgsqlTypes;
+using System;
+using System.Collections.Immutable;
+using System.Data;
+using System.Data.Common;
+using System.Linq;
+using System.Runtime.CompilerServices;
+using System.Text;
+using System.Threading;
+using System.Threading.Tasks;
+using Akka.Event;
+
+namespace Akka.Persistence.PostgreSql.Journal
+{
+    public class PostgreSqlDbBatchQueryExecutor : PostgreSqlQueryExecutor
```

Comment:
Not clear to me - how does this journal get used exactly? Does a user have to specify in the journal HOCON that they want to use this one?

Reply:
Yes, since this places all of the DbBatch dependencies in this class and leave the regular Executor more or less unchanged.

Reply:
Ok, got it.